core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeChecker.scala

```diff
@@ -318,15 +318,17 @@ class PluginTypeChecker(platform: Platform = PlatformFactory.createInstance(),
     (allSupportedReadSources.toMap, allSupportedWriteFormats)
   }
 
-  def getOtherTypes(typeRead: String): Seq[String] = {
+  private def getOtherTypes(typeRead: String): Seq[String] = {
     typeRead match {
-      case "long" => Seq("bigint")
-      case "short" => Seq("smallint")
-      case "int" => Seq("integer")
+      case "binary" => Seq("blob")
       case "byte" => Seq("tinyint")
-      case "float" => Seq("real")
-      case "decimal" => Seq("dec", "numeric")
       case "calendar" => Seq("interval")
+      case "decimal" => Seq("dec", "numeric")
+      case "float" => Seq("real")
+      case "int" => Seq("integer")
+      case "long" => Seq("bigint")
+      case "short" => Seq("smallint")
+      case "string" => Seq("varchar", "char")
```

Comment:
`string` and `binary` are 2 new rows.
The rest of changes in rows were due to order the entries alphabetically.